Marlin/src/module/temperature.cpp

```diff
@@ -286,6 +286,11 @@ uint8_t Temperature::soft_pwm_amount[HOTENDS];
       next_auto_fan_check_ms = next_temp_ms + 2500UL;
     #endif
 
+    if (target > ((heater < 0 ? BED_MAXTEMP : maxttemp[heater]) - 15)) {
```

Comment:
```c++
    #ifdef BED_MAXTEMP

    if (target > ((heater < 0 ? BED_MAXTEMP : maxttemp[heater]) - 15)) {

    #else
    
    if (target > (maxttemp[heater] - 15)) {

    #endif
      SERIAL_ECHOLNPGM(MSG_PID_TEMP_TOO_HIGH);
      return;
    }
```

Reply:
I know, I've seen it, but this will crash if BED_MAXTEMP will be undefined and you will try to do a PID autotune of BED heater